src/Service/PdfCreator.php

```diff
@@ -4,38 +4,39 @@
 
 namespace App\Service;
 
-use Spipu\Html2Pdf\Html2Pdf;
+use Dompdf\Dompdf;
+use Dompdf\Options;
 
 final class PdfCreator implements PdfCreatorInterface
 {
     private $pdf;
     private $orientation;
     private $format;
-    private $lang;
-    private $unicode;
-    private $encoding;
-    private $margin;
 
-    public function __construct(string $orientation, string $format, string $lang, bool $unicode, string $encoding, array $margin)
+    public function __construct(string $orientation, string $format)
     {
         $this->orientation = $orientation;
         $this->format = $format;
-        $this->lang = $lang;
-        $this->unicode = $unicode;
-        $this->encoding = $encoding;
-        $this->margin = $margin;
     }
 
-    public function create(): void
+    public function create(bool $isHtml5ParserEnabled = true, bool $isRemoteEnabled = true, Options $options = null): void
```

Comment:
Looks like `$isHtml5ParserEnabled` and `$isRemoteEnabled` are options after all. Should this be delegated to the caller (and therefore only pass options as parameter, not nullable, hence reducing the state)?